src/pages/_assets/examples-content.tsx

```diff
@@ -0,0 +1,264 @@
+import { PropTypes } from '../../components/Examples/example-list'
+
+export const official: PropTypes = {
+  id: 'official',
+  title: 'Official Examples',
+  description:
+    'Guides, tutorials and configurations for using Ory services. Examples maintained and tested by the Ory team.',
+  examples: [
+    {
+      title: 'Protect a Page with Login: NextJs/React',
+      language: 'typescript',
+      // framework: 'NextJs/React',
+      author: 'ory',
+      tested: true,
+      repo: 'https://github.com/ory/docs/tree/master/code-examples/protect-page-login/nextjs',
+      docs: 'https://www.ory.sh/docs/guides/protect-page-login/next.js'
+    },
+    {
+      title: 'Protect a Page with Login: ExpressJS',
+      language: 'typescript',
+      // framework: 'ExpressJs',
+      author: 'ory',
+      tested: true,
+      repo: 'https://github.com/ory/docs/tree/master/code-examples/protect-page-login/expressjs',
+      docs: 'https://www.ory.sh/docs/guides/protect-page-login/expressjs'
+    },
+    {
+      title: 'Protect a Page with Login: Go',
+      language: 'go',
+      // framework: 'Vanilla',
+      author: 'ory',
+      tested: true,
+      repo: 'https://github.com/ory/docs/tree/master/code-examples/protect-page-login/go',
+      docs: 'https://www.ory.sh/docs/guides/protect-page-login/go'
+    },
+    {
+      title: 'Protect a Page with Login: PHP',
+      language: 'php',
+      // framework: 'Vanilla',
+      author: 'ory',
+      tested: true,
+      repo: 'https://github.com/ory/docs/tree/master/code-examples/protect-page-login/php',
+      docs: 'https://www.ory.sh/docs/guides/protect-page-login/php'
+    },
+    {
+      title: 'Protect a Page with Login: Flutter',
+      language: 'dart',
+      // framework: 'Flutter',
+      author: 'ory',
+      tested: true,
+      repo: 'https://github.com/ory/docs/tree/master/code-examples/protect-page-login/flutter_web_redirect',
+      docs: 'https://www.ory.sh/docs/guides/protect-page-login/-web-redirect'
+    },
```

Comment:
You're using the Dart logo instead of Flutter's, is this intentional?

Reply:
Yea Flutter is the framework and Dart is the language. I tried to be consistent and always use the language logo but maybe its confusing.

Reply:
Makes sense!